pkg/config/app/kuma-dp/config.go

```diff
@@ -150,6 +152,10 @@ type Dataplane struct {
 	DrainTime config_types.Duration `json:"drainTime,omitempty" envconfig:"kuma_dataplane_drain_time"`
 	// Port that exposes kuma-dp readiness status on localhost, set this value to 0 to provide readiness by "/ready" endpoint from Envoy adminAPI
 	ReadinessPort uint32 `json:"readinessPort,omitempty" envconfig:"kuma_readiness_port"`
+	// ResilientComponentBaseBackoff defines the base backoff between restarts of resilient components
+	ResilientComponentBaseBackoff config_types.Duration `json:"resilientComponentBaseBackoff,omitempty" envconfig:"kuma_dataplane_resilient_component_base_backoff"`
+	// ResilientComponentMaxBackoff defines the max backoff between restarts of resilient components
+	ResilientComponentMaxBackoff config_types.Duration `json:"ResilientComponentMaxBackoff,omitempty" envconfig:"kuma_dataplane_resilient_component_max_backoff"`
```

Comment:
shouldn't this be lowercase?

```suggestion
	ResilientComponentMaxBackoff config_types.Duration `json:"resilientComponentMaxBackoff,omitempty" envconfig:"kuma_dataplane_resilient_component_max_backoff"`
```